src/template.html

```diff
@@ -56,6 +56,7 @@
               </a>
             </div>
             <input type="text" class="form-control input-block" data-store="TOKEN" />
+            <div class="octotree-disclaimer-token text-gray text-mono">The access token is stored in your browser local storage and Octotree uses it only to communicate with GitHub API.</div>
```

Comment:
Please don't use GitHub classes (text-gray text-mono) for simple case like this. Just define explicitly the CSS classes.

Change text to "Token is stored in local storage"